restli-server/src/main/java/com/linkedin/restli/internal/server/model/ResourceModelEncoder.java

```diff
@@ -109,6 +116,10 @@
 
   private final DataCodec codec = new JacksonDataCodec();
 
+  // Used to cache the mapping between restspec file and
+  // the class loader that can get the resource
+  private Map<String, ClassLoader> _restSpecToClassLoaderMap = null;
```

Comment:
Would there be any concurrency concerns here? I'm not sure whether the method `loadOrBuildResourceSchema` can be run concurrently or not.

Reply:
There will be no concurreny issue as far as I am aware of. Also I did the testing by single steps and I found the execution is through the loop